internal/proxy/proxy.go

```diff
@@ -34,6 +36,44 @@ import (
 	"golang.org/x/oauth2"
 )
 
+var (
+	// Instance connection name is the format <PROJECT>:<REGION>:<INSTANCE>
+	// Additionally, we have to support legacy "domain-scoped" projects (e.g. "google.com:PROJECT")
+	connNameRegex = regexp.MustCompile("([^:]+(:[^:]+)?):([^:]+):([^:]+)")
```

Comment:
Should we consider just exposing these from the Go connector?

Reply:
Yes, I'll do that.